beamer/tests/agent/unit/test_relayer.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from subprocess import CalledProcessError
+from unittest.mock import patch
+
+from hexbytes import HexBytes
+
+from beamer.agent.relayer import run_relayer_for_tx
+from beamer.tests.agent.unit.utils import make_address, make_context
+
+
+@patch("beamer.agent.relayer.subprocess.run")
+@patch("beamer.agent.relayer.get_relayer_executable")
+def test_redacted_private_key_log(mocked_relayer_executable, mocked_subprocess):
+    context, _ = make_context()
+    fill_tx = HexBytes(make_address())
```

Comment:
Why do we need these? We can just  provide dummy values for both the RPC URL and the fill tx, right?
Since the relayer never gets to be executed, those don't need to be valid anyway.

Reply:
in theory we dont need but i wanted to have a little bit more realistic values but ofc can change